kitchen.dokken.yml

```diff
@@ -30,3 +30,11 @@ platforms:
       intermediate_instructions:
         - RUN /usr/bin/apt-get update
         - RUN /usr/bin/apt-get install sudo -y
+
+  - name: ubuntu-20.04
+    driver:
+      image: dokken/ubuntu-20.04
+      pid_one_command: /bin/systemd
+      intermediate_instructions:
+        - RUN /usr/bin/apt-get update
+        - RUN /usr/bin/apt-get install sudo gcc make -y
```

Comment:
Where is gcc getting required? Seems like this should be in a recipe somewhere...

Reply:
it's for the nokogiri compile which is part of the vagrant-libvirt plugin install, the nokogiri install is really not the point of this cookbook so I don't want it in the example recipe.

Reply:
I don't see a problem with at least including it in the ``test::default`` recipe. Ideally, it would be better to use ``build_essential`` instead IMHO. I think in the long term, it would be better to move all the ``intermediate_instructions`` packages into the test cookbook recipes where needed.

Reply:
Changed.

Reply:
thanks for the review and suggestions.